.github/scripts/generate_binary_build_matrix.py

```diff
@@ -310,12 +324,32 @@ def generate_wheels_matrix(
                         "devtoolset": "",
                         "container_image": WHEEL_CONTAINER_IMAGES[arch_version],
                         "package_type": package_type,
-                        "pytorch_extra_install_requirements": PYTORCH_EXTRA_INSTALL_REQUIREMENTS,
+                        "pytorch_extra_install_requirements": PYTORCH_EXTRA_INSTALL_REQUIREMENTS_CU121,
                         "build_name": f"{package_type}-py{python_version}-{gpu_arch_type}{gpu_arch_version}".replace(  # noqa: B950
                             ".", "_"
                         ),
                     }
                 )
+            elif arch_version == "11.8" and os == "linux":
```

Comment:
Please remove this extra if statement, we would need to have following  if statement: 
```
if (arch_version == "12.1" or arch_version == "11.8")  and os == "linux":
```

Reply:
To simplify it even further:
```python
if arch_version in ["12.1", "11.8"]  and os == "linux"
```